### DISCUSSION TITLE
Cndtool and Matool won't extract

### DISCUSSION THREAD

**Discussion by EdwardProj** [smlu/ProjectMarduk#7] [Q&A] [answered]
Hi, very happy to find modding on my favorite game of all time. When I run either Cndtool or Matool in command prompt or gitbash I get confirmation that it extracts, but I get no output file, even when I specify one, what are you thoughts? Thanks!
![screenshot1](https://github.com/smlu/Urgon/assets/154039151/cba3b596-76dd-4a3f-b8df-06ab6a211d88)

**Comment by smlu** [comment, marked as answer]
Hey, from provided screenshot I see that the current directory of command line is not the `ndy` folder.
Have you maybe checked folder `C:\Users\joepa`?

**Reply by EdwardProj** [reply]
Thank you for the fast response, yes you are correct, I found them there. Before I found them I placed the tools in the C folder and set my root directory to that, I then had no issues. Thank you for creating such a great tool, I've been able to extract files in the past but never been able to modify them and place back in the game.

**Reply by smlu** [reply]
I'm glad you solved it. Btw in cmd if you want to be in specific folder use `cd` command (change directory), 
i.e.: `cd "<directory_path>"`.

**Reply by EdwardProj** [reply]
That's what I did in the end, although for some reason I can only change my root directory to something in the C drive, whereas normally I place all of my data in the D drive. It all worked out in the end.
